tests/llm_providers/test_cohere.py

```diff
@@ -0,0 +1,251 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+
+from nemoguardrails import RailsConfig
+from tests.constants import COHEREAI_API_URL_CHAT
+from tests.utils import TestChat
+
+EXAMPLES_FOLDER = os.path.join(os.path.dirname(__file__), "../../", "examples")
+os.environ['COHERE_API_KEY']='XXXXXXXXX4UsaUWoAk0UYCHcbFaLJAeCjoOpDDrV'
```

Comment:
This is for mock-ups, so we can just put an empty string here, no?
Just for people not to think this is a valid API key.

Reply:
It does not accept empty, but an obviously dummy key works.